reference/api/search.mdx

```diff
@@ -1060,3 +1066,51 @@ The following query returns documents whose `overview` includes `"adventure"`:
 <CodeSamples id="search_parameter_guide_attributes_to_search_on_1" />
 
 Results would not include documents containing `"adventure"` in other fields such as `title` or `genre`, even if these fields were present in the `searchableAttributes` list.
+
+### Hybrid search (experimental)
+
+**Parameter**: `hybrid`<br />
+**Expected value**: An object with two fields: `embedder` and `semanticRatio`<br />
+**Default value**: `null`
+
+Configures Meilisearch to return search results based on a query's meaning and context.
+
+`hybrid` must be an object. It accepts two fields: `embedder` and `semanticRatio`.
+
+`embedder` must be a string indicating an embedder configured with the `/settings` endpoint. If you don't specify an embedder and your index contains a single embedder, Meilisearch uses it by default. If an index contains multiple embedders, Meilisearch will use the embedder named `default`.
+
+`semanticRatio` must be a number between `0.0` and `1.0` indicating the proportion between keyword and semantic search results. `0.0` causes Meilisearch to only return keyword results. `1.0` causes Meilisearch to only return meaning-based results. Defaults to `0.5`.
+
+<Capsule intent="warning">
+Meilisearch will return an error if you use `hybrid` before activating your instance's `vectorStore` and [configuring an embedder](/reference/api/settings#embedders-experimental).
+</Capsule>
+
+#### Example
+
+```sh
+curl -X POST 'localhost:7700/indexes/products/search' \
+   -H 'content-type: application/json' \
+  --data-binary '{ 
+    "q": "kitchen utensils",
+    "hybrid": {
+      "semanticRatio": 0.9,
+      "embedder": "default"
+    }
+  }'
+```
+
+### Vector (experimental)
+
+Use a custom-generated vector to perform a search query. Must be an array of numbers corresponding to the dimensions of the custom vector.
```

Comment:
I feel like we're probably missing some explanations here:

1. `vector` is effectively mandatory for a `userProvided` embedder, as auto-embedding won't work for it
2. `vector` can also be used to punctually override the auto-embedding of the query. If vector is used, then no autoembedding will take place.
3. The custom vector's dimensions must match the dimensions of vectors produced by the embedder, or an error will be returned.